src/lib/browse/layers/points/CriticalIssues.svelte

```diff
@@ -39,19 +39,15 @@
     }
   }
 
-  function clickCluster(e: CustomEvent<LayerClickInfo>) {
-    ($map.getSource(source) as GeoJSONSource).getClusterExpansionZoom(
-      e.detail.features[0].properties!.cluster_id,
-      (err, zoom) => {
-        if (!err) {
-          $map.easeTo({
-            // @ts-ignore We know this is a point
-            center: e.detail.features[0].geometry.coordinates,
-            zoom: zoom as number,
-          });
-        }
-      },
-    );
+  async function clickCluster(e: CustomEvent<LayerClickInfo>) {
```

Comment:
maplibre went through a major internal refactor to use promises instead of callbacks. Nicer APIs as a result :)

Reply:
Is this the only place where it changes stuff for us? Wild that this refactor is happening now but looks good!

Reply:
I mean I guess it must be but it's surprising